### PR TITLE
Add copyright headers to provision module files

### PR DESCRIPTION
**Pull request by QU3B1M**
# Description

This PR adds copyright headers to al the provision module's files.

### CODE REVIEW

File: deployability/modules/provision/playbooks/package/install/install.j2 [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
```
Comment (rauldpm) [inline deployability/modules/provision/playbooks/package/install/install.j2:4]: Remove the copyright header in jinja2 templates, as they will be introduced each time the template is used

**Review by rauldpm — CHANGES_REQUESTED** [review]
Review all j2 files

File: deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml [hunk 2]
```diff
@@ -20,10 +20,10 @@ tasks:
         path: python3
         args:
           - modules/testing/main.py
-          - inventory: "{working-dir}/agent-{agent}/inventory.yaml"
+          - inventory: "{working-dir}/agent-{agent}/inventory.yml"
           - dependencies:
```
Comment (rauldpm) [inline deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml:24]: Keep yaml extension

**Review by rauldpm — CHANGES_REQUESTED** [review]

File: deployability/modules/provision/playbooks/package/install/install.j2 (hunk 1 above)
Comment (QU3B1M) [reply deployability/modules/provision/playbooks/package/install/install.j2]: Done in https://github.com/wazuh/wazuh-qa/pull/5152/commits/98401a0a222ec7cb65be53b354ca3ad4be4b6210

File: deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml (hunk 2 above)
Comment (rauldpm) [reply deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml:24]: Ok, leave it as is, we will standardize this to YAML in tier 2

**Review by rauldpm — APPROVED** [review]

File: deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml [hunk 3]
```diff
@@ -44,9 +44,9 @@ tasks:
         path: python3
         args:
           - modules/testing/main.py
-          - inventory: "{working-dir}/agent-{agent}/inventory.yaml"
+          - inventory: "{working-dir}/agent-{agent}/inventory.yml"
```
Comment (fcaffieri) [inline deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml:47]: Change yml to yaml

**Review by fcaffieri — CHANGES_REQUESTED** [review]

**Review by fcaffieri — APPROVED** [review]

**Pull request merged**